api/backup/restore.go

```diff
@@ -58,12 +59,13 @@ func extractArchive(r io.Reader, destinationDirPath string) error {
 	return archive.ExtractTarGz(r, destinationDirPath)
 }
 
-func restoreFiles(srcDir string, destinationDir string) error {
+func restoreFiles(srcDir string, destinationDir string, databaseFile string) error {
 	for _, filename := range filesToRestore {
 		err := filesystem.CopyPath(filepath.Join(srcDir, filename), destinationDir)
 		if err != nil {
 			return err
 		}
 	}
-	return nil
+
+	return filesystem.CopyPath(filepath.Join(srcDir, databaseFile), destinationDir)
```

Comment:
i guess we can remove `portainer.db` from `filesToRestore`, cannot we?

Reply:
Good point.  Fixing...